src/amo/components/ScreenShots/index.js

```diff
@@ -1,26 +1,13 @@
 /* @flow */
-/* global window */
-import invariant from 'invariant';
 import * as React from 'react';
-import { PhotoSwipeGallery } from 'react-photoswipe';
-import 'react-photoswipe/lib/photoswipe.css';
+import { Gallery, Item } from 'react-photoswipe-gallery';
+import invariant from 'invariant';
```

Comment:
Minor nit: I try to keep these in alphabetical order.

Reply:
thanks, I like that idea a lot!